### ISSUE TITLE
The neck and the skull should be separate "bones".

### ISSUE THREAD

**Issue by 404CoderNotFound** [studio-minus/ppg-bugs-and-requests#1514]
**Feature description**
If you hit someone in the neck, their neck should break (Causing the head to go floppy). If you smack someone on the head really hard, then their skull should fracture (Maybe you could like, make their head cave in a bit? Like in real life?).

**Feature purpose**
Realism. I've always found it irritating that it's not possible to just absolutely smash one's skull.

**Additional context**
Currently, if you were to for example smack someone with a metal bat really hard, either nothing happens or their neck breaks.

**Comment by DindinYT37** [comment]
@Drogo-man lmao

**Comment by YourLocalPlonker** [comment]
Haha imagine a human with a neck. That's silly

**Comment by 404CoderNotFound** [comment]
> @Drogo-man lmao

huh what?

**Comment by Tbskins** [comment]
they could maybe add it if a wire or something is attached to there neck with no slack it slowly drains their oxygen

**Comment by DindinYT37** [comment]
> they could maybe add it if a wire or something is attached to there neck with no slack it slowly drains their oxygen

That literally makes no sense. Why should it be a wire? Also it why should it drain oxygen, you'd be literally dead if the skull is broken and about dead if your neck is broken - not to mention, as far as I know, it doesn't prevent you from getting oxygen if it's broken.

**Comment by Tbskins** [comment]
> > they could maybe add it if a wire or something is attached to there neck with no slack it slowly drains their oxygen
> 
> That literally makes no sense. Why should it be a wire? Also it why should it drain oxygen, you'd be literally dead if the skull is broken and about dead if your neck is broken - not to mention, as far as I know, it doesn't prevent you from getting oxygen if it's broken.

i didn't mean just the wire. I was also just meaning that you would be able to hang people?  btw the way you typed that sounded rude/mean IDK if you meant for it to sound like that

**Comment by DindinYT37** [comment]
> > > they could maybe add it if a wire or something is attached to there neck with no slack it slowly drains their oxygen
> > 
> > 
> > That literally makes no sense. Why should it be a wire? Also it why should it drain oxygen, you'd be literally dead if the skull is broken and about dead if your neck is broken - not to mention, as far as I know, it doesn't prevent you from getting oxygen if it's broken.
> 
> i didn't mean just the wire. I just meant that you would be able to hang people? btw the way you typed that sounded rude/mean IDK if you meant for it to sound like that

It doesn't sound like anything, it sounds how you interpret it.

Also, if you have problems with english (if you're foreign or whatever), I suggest simply saying what you mean instead of explaining it unnecessarily complex - Just adding "so you can hang people" to the end of your comment from before could've prevented this.

**Comment by Tbskins** [comment]
> > > > they could maybe add it if a wire or something is attached to there neck with no slack it slowly drains their oxygen
> > > 
> > > 
> > > That literally makes no sense. Why should it be a wire? Also it why should it drain oxygen, you'd be literally dead if the skull is broken and about dead if your neck is broken - not to mention, as far as I know, it doesn't prevent you from getting oxygen if it's broken.
> > 
> > 
> > i didn't mean just the wire. I just meant that you would be able to hang people? btw the way you typed that sounded rude/mean IDK if you meant for it to sound like that
> 
> It doesn't sound like anything, it sounds how you interpret it.
> 
> Also, if you have problems with english (if you're foreign or whatever), I suggest simply saying what you mean instead of explaining it unnecessarily complex - Just adding "so you can hang people" to the end of your comment from before could've prevented this.

fair enough and i don't have trouble with English just trouble with wording/writing in general, sorry

**Comment by Drogo-man** [comment]
oh no, I HAVE BEEN SUMMONED.
